packages/web/src/components/spa/HackAUBG/VerifyAccountPop/VerifyAccount.jsx

```diff
@@ -1,14 +1,23 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import styles from './verify_account.module.css';
 import { FaRegWindowMinimize } from 'react-icons/fa';
 import { useLocation } from 'react-router-dom';
 import axios from 'axios';
 import { verifyURL } from '../../../../Global';
 
 export const VerifyAccount = ({ className, onSuccess }) => {
+    const [jwtToken, setJwtToken] = useState('');
     const location = useLocation();
-    const params = new URLSearchParams(location.search);
-    const jwtToken = params.get('jwt_token');
+    const [buttonState, setButtonState] = useState(true);
+
+    useEffect(() => {
+        const params = new URLSearchParams(location.search);
+        if (params.size === 0) {
+            window.location.href = '/hackaubg';
```

Comment:
Isn't the url already that? No need to change the value.

Reply:
This is a check if after verify there is a jwt_token. And if there is not and the url looks like this /hackaubg/verify it goes back to /hackaubg

Reply:
okay, makes sense